include/deal.II/cgal/surface_mesh.h

```diff
@@ -66,6 +94,50 @@ namespace CGALWrappers
     const typename dealii::Triangulation<dim, spacedim>::cell_iterator &cell,
     const dealii::Mapping<dim, spacedim> &                              mapping,
     CGAL::Surface_mesh<CGALPointType> &                                 mesh);
+
+  /**
+   * Performs corefinement and boolean operations on two deal.II cells and put
+   * the result in a CGAL::Surface_mesh object. If the cells are disjoint and
+   * you try to perform a boolean operation which makes no sense, like their
+   * intersection, an exception is thrown.
+   *
+   * Extensive examples can be found in the CGAL documentation at
+   * https://doc.cgal.org/latest/Polygon_mesh_processing/index.html#coref_coref_subsec
+   *
+   * @param[in] cell1 First input cell
+   * @param[in] cell2 Second input cell
+   * @param[in] boolean_operation BooleanOperation::INTERSECTION and
+   * BooleanOperation::UNION and BooleanOperation::DIFFERENCE perform
+   * intersection, union and difference of cells, respectively. If not
+   * specified, this parameter is defaulted to BooleanOperation::NONE, and only
+   * a corefinement is performed.
+   * @param[out] outsm Output surface mesh
+   */
+  template <typename CGALPointType, int dim, int spacedim>
+  void
+  boolean_operation(
+    const typename Triangulation<dim, spacedim>::cell_iterator &cell1,
+    const typename Triangulation<dim, spacedim>::cell_iterator &cell2,
+    CGAL::Surface_mesh<CGALPointType> &                         outsm,
+    const BooleanOperation &boolean_operation = BooleanOperation::NONE);
+
+  /**
```

Comment:
You no longer have `spacedim` at your disposal. You should either document what `spacedim` is, or use directly `CGALPointType::Ambient_dimension::value` in place of spacedim in the documentation.